d-rita/fastfoodfast-react
# #164353005 Set up empty react project

#### What does this PR do?
This PR sets up an empty react project. 

#### Description of tasks completed
- Install node and npm
- Add package.json
- Install dependencies
- Set up web pack and babel

#### How to manually test this branch? 
- Fetch and check out this branch using `git fetch origin ch-set-up-react-project-164353005 && git checkout ch-set-up-react-project-164353005`
- To install dependencies, run `yarn install`
- In the terminal, use `yarn start` to run the application on localhost.

#### Related Pivotal Tracker Story 
[#164353005](https://www.pivotaltracker.com/story/show/164353005)
